crosslink/cmd/root.go

```diff
@@ -123,14 +134,16 @@ func Execute() {
 }
 
 func init() {
-
 	comCfg.rootCommand.PersistentFlags().StringVar(&comCfg.runConfig.RootPath, "root", "", `path to root directory of multi-module repository. If --root flag is not provided crosslink will attempt to find a
 	git repository in the current or a parent directory.`)
-	comCfg.rootCommand.PersistentFlags().StringSliceVar(&comCfg.excludeFlags, "exclude", []string{}, "list of comma separated go modules that crosslink will ignore in operations."+
-		"multiple calls of --exclude can be made")
 	comCfg.rootCommand.PersistentFlags().BoolVarP(&comCfg.runConfig.Verbose, "verbose", "v", false, "verbose output")
+	comCfg.rootCommand.Flags().StringSliceVar(&comCfg.excludeFlags, "exclude", []string{}, "list of comma separated go modules that crosslink will ignore in operations."+
```

Comment:
Why not support this with the work command?

Reply:
In my opinion, the `exclude` flag for `work` command should accept "paths" instead of "Go module names" as. This is because is `go.work` has no coupling the Go module names. It only operates on paths. Because it is different, I decided not adding this flag until it is needed.

Of course, we would need to parse the `go.mod` files if we would like to support the `exclude` as "Go module names". However, I really think that using "path" would be better".